packages/tools/client-debugger/client-debugger-view/src/components/ValueView.tsx

```diff
@@ -19,20 +21,16 @@ export interface ValueViewProps {
 export function ValueView(props: ValueViewProps): React.ReactElement {
 	const { node } = props;
 
-	const stackStyles: IStackStyles = {
-		root: {
-			padding: "10px",
-			background: "rgb(237, 235, 233)",
-		},
-	};
-
 	return (
-		<Stack>
-			<StackItem styles={stackStyles}>
-				{`${node.metadata !== undefined ? `${node.metadata} : ` : ""}
+		<Tree>
```

Comment:
I suspect we don't need to use the `Tree` components for the value-type views, since they don't display any children.

Reply:
Removed.